toolkit/docs/formats/imageconfig.md

```diff
@@ -111,6 +111,11 @@ A sample PackageLists entry pointing to three files containing package lists:
     "packagelists/cloud-init-packages.json"
 ],
 ```
+### RemoveRpmDb
+
+RemoveRpmDb triggers RPM database removal after the packages have been installed.
+Removing RPM database can put system in an invalid state. This is only recommended for images that do not contain package manager, such as some containers.
```

Comment:
Suggestion: `Removing the RPM database can put the system in an invalid state if its expected to use a package manager`.

Currently it sounds like this option may cause image corruption.